### ISSUE TITLE
tvOS does not support tessellated drawing from indirect buffers

### ISSUE THREAD

**Issue by billhollings** [KhronosGroup/Vulkan-Portability#5]
Specifically...the `drawPatches:...indirectBuffer:...` style methods are not available.

@cdavis5e I'll defer to your implementation expertise on this one.

Do we need a portability flag for this? Or is a workaround possible?

@kvark commented: Does tvOS even have GPU-local memory? If it doesn't we can just read the contents of the `indirectBuffer` on CPU and behave accordingly.

**Comment by cdavis5e** [comment]
> Does tvOS even have GPU-local memory? If it doesn't we can just read the contents of the `indirectBuffer` on CPU and behave accordingly.

No, but if the buffer has `Private` storage, Metal won't let us read its `contents`. I guess if the usage sets `INDIRECT_BUFFER_BIT`, we could override it to `Shared`.
